src/gui_common/TweakedColourPicker.cs

```diff
@@ -222,6 +227,15 @@ public override void _Ready()
         presetsContainer = GetNode<GridContainer>("PresetContainer");
         addPresetButton = GetNode<TextureButton>("PresetButtonContainer/AddPresetButton");
 
+        // Picker button logic rewrite #3055
+        // TODO: Revert this PR once https://github.com/godotengine/godot/issues/57343 is solved.
+        baseControl = GetChild(1);
+        var customPickerButton = new ToolButton { Icon = pickerButton.Icon };
+        baseControl.RemoveChild(pickerButton);
```

Comment:
Is it possible to overwrite the default connection (so unconnect first) and then just link us to the default button? That would mean we mess with Godot's internals less here.

Reply:
I guess we don't know the connected object and method name so we can't actually call `Disconnect` here.